frame/staking/src/pallet/impls.rs

```diff
@@ -1603,28 +1603,27 @@ impl<T: Config> StakingInterface for Pallet<T> {
 		Self::nominate(RawOrigin::Signed(ctrl).into(), targets)
 	}
 
-	#[cfg(feature = "runtime-benchmarks")]
-	fn nominations(who: Self::AccountId) -> Option<Vec<T::AccountId>> {
-		Nominators::<T>::get(who).map(|n| n.targets.into_inner())
-	}
+	sp_staking::runtime_benchmarks_enabled! {
```

Comment:
This is more verbose now, what's the benefit?

Reply:
The `sp-staking` crate has the `runtime-benchmarks` feature. When this feature is enabled, it will require that this extra function is implemented. 

Before you required that someone also enabled the `runtime-benchmarks` feature for `pallet-staking` to have the function implemented. With this macro it would be different as the function will be implemented by `pallet-staking` the moment the `runtime-benchmarks` feature is enabled for `sp-staking`. This is actually cleaner and we should use this approach more often. For example for `try-runtime`. You currently require that every pallet has this feature enabled, while they maybe not even do anything with this feature besides implementing the `Try*`  trait that is done by the macro. However, it would be better to have the `Try*` implemented when the `try-runtime` feature is enabled for `frame-support`.